tests/tests/libui-rs/simple/output.txt

```diff
@@ -38,11 +36,11 @@ try
 [1;34m  -->[0m src/controls.rs:373:5-378:6
 
 try this
-[0]: [1;33mSuggestion - Replace:[0m
+[0]: [1;33mAt:[0m
+[31m⍉[0mpub fn new() -> Entry {
```

Comment:
Fun bug, but this is a clippy bug, so I won't fix it here. Clippy should be placing this suggestion properly.